examples/provisioner/bottlerocket.yaml

```diff
@@ -23,7 +23,7 @@ spec:
     - deviceName: /dev/xvda
       ebs:
         volumeType: gp3
-        volumeSize: 2Gi
+        volumeSize: 2G # if using GPU, replace with 4G since the GPU bottlerocket AMI requires at least 4G
```

Comment:
Gi is the correct format, right?
the typo is in the one with 20GiBs

Reply:
They both work, G is a power of 10 version and Gi is the power of 2 version. https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory

Reply:
right... but helps if we align with what AWS EC2 and AMIs provide.
If EBSs with 20 gigabytes are 20Gi, that is what I feel we should use, since if we were to use 20G, the volume would be smaller than the AMI and fail.

Similar for RAM. If an ec2 has 16 gigabytes of RAM, using 16G is better than Gi ?

and yeah, I'm aware that HDD and SSD use base 2 and base 10 each.... _OEMs_ ....